src/shared/tpm2-util.c

```diff
@@ -381,16 +383,81 @@ static int tpm2_make_primary(
         return 0;
 }
 
+static void tpm2_pcr_mask_to_selecion(uint32_t mask, uint16_t bank, TPML_PCR_SELECTION *ret) {
+        assert(ret);
+
```

Comment:
should there be some asserts that bank/mask have sensible values too, to be sure?

Reply:
i'll add one for the mask, to encode it must be 24bit at max, because the code can't deal with more (and there shouldn't be more). But for this low-level function the bank actually doesn't matter, we just store in 1:1, hence I don't think we need to validate it further in this internal, minor helper, the caller should have done that already (and does)